src/msg/async/Stack.h

```diff
@@ -235,8 +248,10 @@ class Worker {
   Worker(CephContext *c, unsigned worker_id)
     : cct(c), perf_logger(NULL), id(worker_id), references(0), center(c) {
     char name[128];
-    sprintf(name, "AsyncMessenger::Worker-%u", id);
+    char name_prefix[] = "AsyncMessenger::Worker";
+    sprintf(name, "%s-%u", name_prefix, id);
     // initialize perf_logger
+
```

Comment:
If you want to stick a blank line here, leave the comment adjacent to the thing it describes:
```
    sprintf(name, "%s-%u", name_prefix, id);

    // initialize perf_logger
    PerfCountersBuilder plb(cct, name, l_msgr_first, l_msgr_last);
```